multinet/downloaders/csv.py

```diff
@@ -31,13 +26,15 @@ def download(workspace: str, table: str) -> Any:
     `workspace` - the target workspace
     `table` - the target table
     """
-    space = get_workspace_db(workspace)
-    if not space.has_collection(table):
+    loaded_workspace = Workspace(workspace)
+    if not loaded_workspace.has_table(table):
         raise NotFound("table", table)
 
-    limit = workspace_table_row_count(workspace, table)
-    table_rows = workspace_table_rows(workspace, table, 0, limit)
-    fields = workspace_table_keys(workspace, table, filter_keys=True)
+    loaded_table = loaded_workspace.table(table)
+    limit = loaded_table.row_count()
+    table_rows = loaded_table.rows(0, limit)["rows"]
```

Comment:
Can't you just call `.rows()` without the `limit` parameter and get all the rows? Having to call `.row_count()` first seems like bad design.

Reply:
Yeah, I'll make this change. This is vestigial from the previous implementation, not sure why I didn't notice this.